programs/drift_vaults/src/instructions/update_vault.rs

```diff
@@ -46,11 +46,6 @@ pub fn update_vault<'info>(
     }
 
     if let Some(hurdle_rate) = params.hurdle_rate {
-        validate!(
-            hurdle_rate < vault.hurdle_rate,
-            ErrorCode::InvalidVaultUpdate,
-            "new hurdle rate must be less than existing hurdle rate"
-        )?;
```

Comment:
shouldn't the hurdle rate be up only in order to benefit depositors?
without this, would it lead to retroactive profit charges?